assets/stylesheets/common/common.scss

```diff
@@ -1012,6 +806,10 @@ body.has-full-page-chat {
     }
   }
 
+  .chat-channel-title {
```

Comment:
This is affecting chat channel titles in the sidebar. The spacing is now fairly large. We will need to only target the titles for the info screen here.